### ISSUE TITLE
Integration can't load with recent 2023.6 Version

### ISSUE THREAD

**Issue by ChristophCaina** [mvdwetering/yamaha_ynca#172]
**Describe the bug**
It seems that the Integration can't load with the new 2023.6 Update.

![Screenshot_20230608-014924](https://github.com/mvdwetering/yamaha_ynca/assets/26391061/11b065e0-97d8-478a-a2be-e89ec861b919)
![Screenshot_20230608-014944](https://github.com/mvdwetering/yamaha_ynca/assets/26391061/a7c956c0-3270-4289-9cbe-f3c5714b9843)

```
Logger: homeassistant.config_entries
Source: config_entries.py:1247
First occurred: 01:48:54 (1 occurrences)
Last logged: 01:48:54

Config entry 'RX-V477' for yamaha_ynca integration not ready yet: Could not initialize YNCA receiver RX-V477; Retrying in background
```

**To Reproduce**

**Expected behavior**

**Diagnostics**

**Logging**
If there are errors in the Home Assistant logs please add them here.

**Home Assistant version information:**
 - Version [e.g. 2022.8.0]

**Additional context**

**Comment by ChristophCaina** [comment]
Maybe related with #125

**Comment by friedpenguin** [comment]
Sad but true. Borked as of today. Can't reconfigure.

**Comment by felalex** [comment]
I can confirm the same.

I've removed the integration and then tried to add again.

Got error that can't connect "Failed to connect, check settings."

No erros in log...

**Comment by swampdogmash** [comment]
I can also confirm the same. The integration says that it cannot connect, yet the receiver is on, i can ping it from the home-assistant box, I can telnet to port 50000 from the home assistant box, and they are in the same subnet with no firewall - needless to say, nothing has changed.

Let me know if you want me to test anything.

**Comment by swampdogmash** [comment]
Please find below a link to a tcpdump taken from the homeassistant box seeing the issue.

192.168.5.12 is the Receiver
192.168.5.6 is the home assistant box

subnet is /17

https://1drv.ms/u/s!AqzYD5ChAxGXkC_Q3mQKjGU41sdP?e=I0k0lj

**Comment by ChristophCaina** [comment]
Seems to Work with current dev branch :)
![Screenshot_20230608-232136](https://github.com/mvdwetering/yamaha_ynca/assets/26391061/3557ec7f-4e7b-4a56-b5ac-5cd19ac46ecc)

**Comment by mvdwetering** [comment]
I just released 7.3.0, but it is still marked as a pre-release since there were some hiccups and had to disable some automated checks. I am 99% sure it works because of the manual tests I did, but would like someone to test also.

When installing through HACS you will have to select redownload and make sure the Show beta versions is enabled.

If I get confirmation that it works I can promote it to a normal release.

**Comment by Sn00kiT** [comment]
> I just released 7.3.0, but it is still marked as a pre-release since there were some hiccups and had to disable some automated checks. I am 99% sure it works because of the manual tests I did, but would like someone to test also.
> 
> When installing through HACS you will have to select redownload and make sure the Show beta versions is enabled.
> 
> If I get confirmation that it works I can promote it to a normal release.

Thanks mate for your awesome work on this and your quick response to the problem. I can confirm it works again with 7.3.0

**Comment by mvdwetering** [comment]
Thanks for the confirmation, promoted 7.3.0 to normal release.

**Comment by swampdogmash** [comment]
Thanks, incredibly quick response time! Can confirm after "redownloading" 7.3.0 that it is working with my RX-V677. Brilliant, thank you :)

**Comment by mvdwetering** [comment]
Closing since it is fixed with release of 7.3.0
